packages/binding-coap/src/coap-client.ts

```diff
@@ -44,9 +44,6 @@ export default class CoapClient implements ProtocolClient {
         // WoT-specific content formats
         coap.registerFormat(ContentSerdes.JSON_LD, 2100);
         // TODO also register content fromat with IANA
```

Comment:
typo: fromat

Reply:
I now simply removed these lines as the registration of `application/td+json` is already done :) Or do you think this is still a TODO for the WoT Working Group?

Reply:
This is more a question of the working group rather than node-wot. Hence I think it is ok!